lib/user_history.rb

```diff
@@ -0,0 +1,52 @@
+module UserHistory
```

Comment:
Just realized: We do have a readings history feature, so maybe calling it KinHistory/KinLogs will be better?